blogs/whats_new_1.6/index.md

```diff
@@ -0,0 +1,225 @@
+
+1. Option to alert if it's dropping flows<br>
+There is a new option in FlowCollector to control whether an alert is shown if it's dropping flows (Figure 6).  By default, this is enabled.
+
+![Disable alerts](images/ebpf_metrics_configuration.png)
+_<div style="text-align: center">Figure 6: eBPF Agent configuration - Disable alerts</div>_
+
+2. Use enhanced TCx hook<br>
+Internally, the eBPF Agent uses the Traffic Control (TC) hook to probe the ingress and egress traffic on the interfaces.  In OCP 4.16, which upgrades RHEL to 9.4, it can leverage the enhanced TCx hook for performance gains.
+
+3. Always save DNS flags<br>
+If the *DNSTracking* feature is enabled, regardless of the sampling setting, it always saves the DNS id and flags.  Previously, it would discard this data if the flow didn't exist.
+
+
+## Flow Collector Enhancements
+
+A few enhancements were made in Flow Collector.  They are:
+
+1. Filter on not 'ip'<br>
+In **Observe > Network Traffic**, for the IP-related filter fields, such as SourceIP, Destination IP, Source Node IP, and Destination Node IP, you can filter by *not* matching or excluding that IP or range (Figure 7).
+
+![Not 'ip' filter](images/not_ip_filter.png)
+_<div style="text-align: center">Figure 7: Not 'ip' filter</div>_
+
+2. De-duplication of flows<br>
+    When a flow is a duplicate on another interface, instead of saving multiple copies of that flow data, only one is stored now.  It keeps track of the list of interfaces that refer to the flow data.  This greatly reduces the amount of storage.
+
+    The raw JSON fields have been renamed and pluralized to `Interfaces` and `IfDirections`, since the type changed to a list from a single value.  Note the use of square brackets for the lists below.
+
+    ```
+      "IfDirections": [
+        0
+      ],
+      "Interfaces": [
+        "br-ex"
+      ],
+    ```
+
+3. Subnet labels<br>
+    You can define a list of names for CIDRs (IP ranges and/or hosts).  When the CIDR matches, the name appears in the flow data as the `SrcSubnetLabel` or `DestSubnetLabel` field.
+
+    YAML Example:
+
+    ```
+      processor:
+        subnetLabels:
+          customLabels:
+          - cidrs:
+            - 10.129.0.73/32  # provide a name for this host
+            name: querier
+          openShiftAutoDetect: false
+    ```
+
+    If you don't want to define names, you can enable the OpenShift auto-detect mode to identify the items as "Pods", "Services", "Machines" or "n/a" (not applicable).  You configure this in the Processor section of Flow Collector (Figure 8).
+
+![Subnet labels configuration](images/subnet_labels_configuration.png)
+    _<div style="text-align: center">Figure 8: Subnet labels configuration</div>_
+
+   Figure 9 shows the subnet label columns in the Traffic flow table when OpenShift auto-detect is enabled.
+
+![Subnet labels columns](images/subnet_labels_columns.png)
+    _<div style="text-align: center">Figure 9: Subnet labels columns</div>_
+
```

Comment:
you can also mention that this subnet feature provides a reliable way to identify cluster-external traffic, by matching anything that is not labelled.

Also, we may give some good practice here, for instance if users want to label their own external workloads, they could use a common prefix for the labels such as "Ext_" (e.g. Ext_MySQL, Ext_S3), so that they can easily count that in external traffic, for instance in promQL:

```
# All cluster-egress:
sum(rate(netobserv_workload_egress_bytes_total{DstSubnetLabel=~"|Ext_.*"}[1m]))
# Same, by configured labelled:
sum(rate(netobserv_workload_egress_bytes_total{DstSubnetLabel=~"|Ext_.*"}[1m])) by (DstSubnetLabel)
```

(this is assuming `workload_egress_bytes_total` metrics has been enabled in `processor.metrics.includeList`)

![Capture d’écran du 2024-06-28 09-53-08](https://github.com/netobserv/documents/assets/2153442/41b303c3-2b34-46c4-a639-820e0e716315)